pkg/gzip/options.go

```diff
@@ -8,29 +8,33 @@ import (
 	"errors"
 	"flag"
 	"fmt"
-	"os"
 	"path/filepath"
 	"runtime"
 
 	"github.com/klauspost/pgzip"
 )
 
+var (
+	ErrStdoutNoForce = errors.New("standard output is a terminal -- ignoring")
```

Comment:
Nit: Ignoring what? Also, the text is maybe a bit long? Could it be 
ignoring --force for stdout
of something? I'm just wondering about people searching logs.

Reply:
"standard output is a terminal -- ignoring" is actually an bsd gzip error, linux have a slightly different error: `gzip: compressed data not written to a terminal. Use -f to force compression.` I will update to something shorter.